pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -438,6 +438,20 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 		},
 	}
 
+	// Add liveness probe if necessary
+	if _, ok := c.healthCheck.LivenessProbe[cephv1.KeyOSD]; ok {
```

Comment:
Adding the liveness probe looks like another place where a helper would be really helpful for common code.

Reply:
I'll look into this too

Reply:
Done